content/2-templating/2-styling/angular/cssstyle.component.ts

```diff
@@ -2,7 +2,16 @@ import { Component } from '@angular/core';
 
 @Component({
 	selector: 'app-cssstyle',
-	templateUrl: './cssstyle.component.html',
-	styleUrls: ['./cssstyle.component.css'],
+	template: `
+		<h1 class="title">I am red</h1>
+		<button style="font-size: 10rem">I am a button</button>
+	`,
+	styles: [
+		`
+			.title {
+				color: red;
+			}
+		`,
+	],
```

Comment:
What bothers me is that in a real world Angular app, HTML & CSS are in separated files.
Obviously the snippet is little. You also loose syntax highlighting. 
The purpose of Component Party is also showing real world examples.

Reply:
In fact, in real world Angular apps, if the components are small, most of the time they exist in single file.

For syntax highlighting I opened an issue to check if we can add it for Angular only.